### DISCUSSION TITLE
Freeplay and Special Thanks not working when building.

### DISCUSSION THREAD

**Discussion by ghost** [luckydog7/Funkin-android#1680] [Q&A]
After building this game to be used in HTML5 and for Windows it seems Freeplay doesn't want to work nor does the Special Thanks tab, not sure if it's a bug with icons or the music, an answer for this would be really helpful.

**Comment by KlavierGayming** [comment]
simple answer
in FreeplayState.hx remove the line that has "NGio.logEvent("Fresh");"
do this again for the credits
